andela/ah-the-phoenix
# #163388133 Implement user email verification

#### What does this PR do?
- Customize the API's landing page to suit an API style
- Added an is_verified field on User model
- Create a HTML template for email message
- Add code to draft and send email
- Add code to verify user token from verification link

#### Description of Task to be completed
- Customize the API's landing page to suit an API style
- Added an is_verified field on User model
- Create a HTML template for email message
- Add code to draft, send and verify email

#### How should this be manually tested?
1. Clone the repo
   `git clone https://github.com/andela/ah-the-phoenix.git`
2. Change it into the current directory
   `cd ah-the-phoenix`
3. Run tests
   `python manage.py runserver`
4. Register new user on _http://127.0.0.1:8000/api/v1/users/_
Sample Data: 
```
{
  "user": {
    "email": "sample@gmail.com",
    "password": "mypassword",
    "username": "Jacob Maze"
  }
}
```
Sample output: 
![image](https://user-images.githubusercontent.com/42174387/52283704-08143d00-2974-11e9-95c6-dcde867322cf.png)


5. Check email inbox for verification link
Email format:
![image](https://user-images.githubusercontent.com/42174387/52266102-9a055100-2946-11e9-9867-ad0caad54a10.png)


#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
 #163388133
